Python/bytecodes.c

```diff
@@ -2174,6 +2174,9 @@ dummy_func(
         };
 
         inst(ENTER_EXECUTOR, (--)) {
+            CHECK_EVAL_BREAKER();
+            JUMPBY(1-oparg);
```

Comment:
This confuses me. Here it appears `oparg` is the jump target, like for the `JUMP_BACKWARD` opcode this is most likely replacing. OTOH three lines below, `oparg` is an index in the `executors` array. I think the latter interpretation is correct (that's what the `_PyOptimizer_BackEdge()` does. It looks like you really ought to inspect the `vm_data` field of the executor for the jump offset.